broadbean/sequence.py

```diff
@@ -601,18 +619,18 @@ def SR(self):
         return SR
 
     @property
-    def channels(self):
+    def channels(self) -> Optional[List]:
```

Comment:
never returns None, and can we say list of what is returned?
```suggestion
    def channels(self) -> List[?]:
```